pyproject.toml

```diff
@@ -13,6 +13,7 @@ readme = "README.md"
 requires-python = ">=3.10"
 dependencies = [
     "pandas<3.0.0",
+    "pyarrow>=21.0.0",  # for parquet content-defined chunking
```

Comment:
We don't need to add this dependency to the core trackio library, as `export_to_parquet()` is only called from Spaces, and we already automatically install `pyarrow` when Trackio is running on Spaces here: https://github.com/gradio-app/trackio/blob/137e0eaf52da21993ec2d693f6c1777897dbd61e/trackio/deploy.py#L103

That being said, we can probably clean this logic up a bit.